subprojects.md

```diff
@@ -0,0 +1,16 @@
+# Subprojects
+
+## Perses
+
+The first subproject is [Perses](https://github.com/perses/perses). Perses
+provides a secure way to configure Prometheus, and we are working toward
+providing visualizations for Prometheus through a new dashboard definition. We
+welcome contributions to this project.
```

Comment:
@Nexucis Let me know if you'd like to make any changes to this description of Perses. I grabbed what looked like a concise summary and combined it with the text about visualizations coming soon.

Reply:
I think that's still an old text from when Perses was intended to do more than dashboarding :)

I would say something like: "Perses is aiming to become a dashboard visualization tool for Prometheus and other datasources. It will focus on being GitOps-compatible via a new and well-defined dashboard definition model."

Reply:
ah yeah sorry about that @geekygirldawn. I forgot to update the readme of Perses according to the last decisions taken.

The description of @juliusv is more accurate :). If we want to give more details, we can say it will be k8s natif. It will provide a way to support the dashboard as service. This kind of thing :)

Reply:
Done! I've updated the description with what Julius suggested just to keep it simple. @Nexucis - it would be great if you could add the details about being k8s native and support for dashboard as a service when you update the Perses readme!

Reply:
Cool thank you !! Yeah I will update it once back from holidays, with that and probably with:
* the matrix channel
* probably a bit of how you can contribute to Perses 
* the docs from here ( code of conduct ..etc.)
* and the dco which is not yet activated